### PR TITLE
Feature/whennext

### PR DESCRIPTION
**Pull request by JR-Mitchell**
## What?
Adds a small cog for setting up a simple vote for the date to have an event on

## Why?
So that it's easy to vote for a date

## Anything else?
This could have many more features, this is just the simplest useful form I could think of.

### CODE REVIEW

**Comment by fjebaker** [comment]
Some linter fixes in #43 @JR-Mitchell if you could just fix the linter issues relevant to your cog

File: src/cogs/when_next.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        while from_date < until_date:
+            if from_date.weekday() > 4:
+                weekend_days.append(from_date.strftime(DATE_FORMAT))
+            from_date = from_date + datetime.timedelta(days=1)
+        ipl = InteractionPipeline(
+            ChoiceInteraction(*weekend_days)
+        )
+        await ipl.send_and_watch(
+            context.channel,
+            discord.embed(
```
Comment (fjebaker) [inline src/cogs/when_next.py]:
```
************* Module cogs.when_next
src/cogs/when_next.py:52:12: E1101: Module 'discord' has no 'embed' member; maybe 'Embed'? (no-member)
```

Comment (JR-Mitchell) [reply src/cogs/when_next.py]: All linting issues fixed as of 1324b2b

**Comment by fjebaker** [comment]
LGTM

**Pull request merged**